server/lib/report-manager.js

```diff
@@ -101,7 +99,7 @@ class ReportManager
         subject = $t('server.email.report.subject', { frequency });
         subject = `Kresus - ${subject}`;
 
-        let today = moment().format('DD/MM/YYYY');
+        let today = moment().format('L');
```

Comment:
Does this work? As `moment` is already an instance of the lib `moment`, I don't know whether we can again call it to create new instances based on the parent...